scripts/scripts.js

```diff
@@ -142,9 +142,12 @@ export function decorateMain(main) {
   decorateVideoLinks(main);
 }
 
-function decorateTemplateAndTheme() {
+// if it gets a block as argument, treat that as specific styling
+// only for that block
+// otherweise style the entire body
+export function decorateTemplateAndTheme(block = document.body) {
```

Comment:
If you need the light/dark decoration, can you instead extrac the logic to another function (such as "decorateMode(element)")? This way you'll avoid calling "aemDecorateTemplateAndTheme" in the blocks.